### PR TITLE
Update cesm2-le catalog based on Steve's comments

### PR DESCRIPTION
**Pull request by mgrover1**

### CODE REVIEW

File: catalogs/glade-cesm2-le.json [hunk 1]
```diff
@@ -115,5 +115,5 @@
   "esmcat_version": "0.0.1",
   "id": "glade-cesm2-le",
   "description": "ESM collection for the CESM2 LENS data stored on GLADE in /glade/campaign/cgd/cesm/CESM2-LE/timeseries",
-  "last_updated": "2021-12-15T20:25:10+00:00"
+  "last_updated": "2021-01-19T20:25:10+00:00"
```
Comment (mnlevy1981) [inline catalogs/glade-cesm2-le.json:118]: It's 2022 :)

Comment (mgrover1) [reply catalogs/glade-cesm2-le.json:118]: Great catch :)

Comment (andersy005) [reply catalogs/glade-cesm2-le.json]: Interesting... are you updating this manually or Is this a bug in ecgtools ( I thought ecgtools was updating this field accordingly)?

Comment (mgrover1) [reply catalogs/glade-cesm2-le.json]: manually... didn't regenerate the entire catalog

**Review by andersy005 — APPROVED** [review]

**Pull request merged**